src/popup/utils/constants.ts

```diff
@@ -68,6 +72,69 @@ export const TX_FUNCTIONS = {
   claim: 'claim',
 } as const;
 
+export const TX_FUNCTION_TYPE_MULTISIG = {
+  propose: 'propose',
+  confirm: 'confirm',
+  refuse: 'refuse',
+  revoke: 'revoke',
+} as const;
+
+export const TX_FUNCTION_TYPE_DEX: Record<DexFunctionType, TxFunctionRaw[]> = {
+  pool: [
+    'remove_liquidity', 'remove_liquidity_ae', 'add_liquidity', 'add_liquidity_ae',
+  ],
+  removeLiquidity: [
+    'remove_liquidity', 'remove_liquidity_ae',
+  ],
+  addLiquidity: [
+    'add_liquidity', 'add_liquidity_ae',
+  ],
+  swap: [
+    'deposit', 'withdraw', 'swap_exact_tokens_for_tokens', 'swap_tokens_for_exact_tokens',
+    'swap_exact_ae_for_tokens', 'swap_tokens_for_exact_ae', 'swap_exact_tokens_for_ae',
+    'swap_ae_for_exact_tokens',
+  ],
+  allowance: [
+    'transfer_allowance', 'change_allowance', 'create_allowance',
+  ],
+  maxSpent: [
+    'swap_tokens_for_exact_tokens', 'swap_tokens_for_exact_ae', 'swap_ae_for_exact_tokens',
+  ],
+  minReceived: [
+    'swap_exact_tokens_for_tokens', 'swap_exact_ae_for_tokens', 'swap_exact_tokens_for_ae',
+  ],
+};
+
+/**
+ * ITx.tag
+ */
+export const TX_TAGS_SUPPORTED = [
+  Tag.SpendTx,
+  Tag.ContractCreateTx,
+  Tag.ContractCallTx,
+  Tag.NamePreclaimTx,
+  Tag.NameClaimTx,
+  Tag.NameUpdateTx,
+  Tag.NameTransferTx,
+];
+
+export const TX_NAME_TAGS = new Set([
+  Tag.NameClaimTx,
+  Tag.NamePreclaimTx,
+  Tag.NameRevokeTx,
+  Tag.NameUpdateTx,
+]);
+
+export const TX_RETURN_TYPE_OK = 'ok';
+export const TX_RETURN_TYPE_ABORT = 'abort';
+export const TX_RETURN_TYPE_REVERT = 'revert';
+
+export const TX_RETURN_TYPES = [
+  TX_RETURN_TYPE_OK,
+  TX_RETURN_TYPE_ABORT,
+  TX_RETURN_TYPE_REVERT,
+] as const;
+
```

Comment:
IDEM. The naming of the constants could be improved.

Reply:
Yeah, I already improved the naming :D